general/src/processor/funcs.rs

```diff
@@ -11,12 +11,6 @@ pub fn process_value<T, P>(
     T: ProcessValue,
     P: Processor,
 {
+    processor.before_process(annotated.0.as_mut(), &mut annotated.1, state);
```

Comment:
Props to @jan-auer here for making `process_child_values` invoke `funcs::process_value` instead of `ProcessValue::process_value`!!